### DISCUSSION TITLE
It's possible the explosion effect?

### DISCUSSION THREAD

**Discussion by TheGuyFawkes** [onlaj/Piano-LED-Visualizer#520] [Ideas]
First off, I just want to say thanks for this amazing application. I’ve been using it, and I really appreciate all the effort you’ve put into it. I had an idea that I think could make the experience even better.

Right now, when I hit a note, three LEDs light up and turn off at the same time (the central note and two adjacent ones). I thought it would be awesome if we could customize this a bit more. Here are a few ideas:

It would be great if we could choose how many LEDs light up on each side of the central note (maybe 2, 3, or more).

Instead of all lighting up at once, how about having them light up progressively? Starting from the central note and moving toward the outer LEDs, kind of like an expanding wave. And then they could turn off in the same order.

It would also be nice to adjust the color of the LEDs on the sides to make it look even cooler.

Oh, and I think we could call it the "explosion effect" or "expanding wave." I’m sure this effect would look really nice.

Could you consider this for future versions? I think a lot of users would love to have more options to customize the effects to match their style.

Thanks for your time!  I really appreciate you sharing this project with us.

**Comment by onlaj** [comment]
Hey, thank you!
Pulse/wave effect is on my to-do list. I can't promise when it will be added, but I'll definitely work on it.

**Reply by TheGuyFawkes** [reply]
Thanks for your reply! I'll be looking forward to the implementation. 
Anyway, I'm going to try to create some functions in the code to include it. I haven't programmed in python for a long time but I think I can do something with the help of AI.
If I manage to get it working, I'll pass it on to you so you can take a look and keep it in mind, what do you think? 

In addition to this, I have a problem when connecting the visualizer with synthesia (on an android tablet) for the learning mode. I'm going to open a new post with the question so that it is organized. 

Thanks for everything!
